repository/SmalltalkCI-Pharo-Core.package/SCIPharoCodeCoverage.class/methodProperties.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
smalltalkCI does not (yet) use metadataless Filetree for compatibility reasons. Could you please restore the metadata files?